discovery-provider/src/tasks/entity_manager/entities/developer_app.py

```diff
@@ -196,6 +197,23 @@ def validate_developer_app_tx(params: ManageEntityParameters, metadata):
             raise Exception(
                 "Invalid Create Developer App Transaction, description must be under 161 chars"
             )
+
+        num_existing_apps_from_user = (
+            session.query(DeveloperApp).filter(DeveloperApp.user_id == user_id).count()
```

Comment:
can this come from existing_records instead of querying here? that's the typical pattern for validations

Reply:
oooo yeah i can do that! what's the reasoning though? feels like this is more straightforward than adding a "developer apps by user" entity and putting that in existing records (+ would have to remember to add to it when a developer app is created). is it to avoid repeated queries?
i also saw in some other files (e.g. tracks, users) the query is just done in the validator like this

Reply:
ah yea you're right, i thought it'd already had all the developer apps from a user

Reply:
o0o0oo0